main.js

```diff
@@ -0,0 +1,49 @@
+const opts = {
+    channels: [
+        twitchChannel
+    ]
+};
+
+let actionHandlers = {};
+let allHandlers = [];
+
+// Create a client with our options defined at the top of the file
+let client = new tmi.client(opts);
+
+// Register our event handlers (defined below)
+client.on('message', onMessageHandler);
+client.on('connected', onConnectedHandler);
+
+// Connect to Twitch:
+client.connect();
```

Comment:
Think if you move these after the function is defined the functions can become `const`